### PR TITLE
Added my info

### PR DESCRIPTION
**Pull request by hossainalikram**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -431,6 +431,24 @@ <h1 class="person__name">Amr Ramadan</h1>
                         </section>
                     </div>
                 </div>
+				<div class="sympl onethird">
```
Comment (roonyh) [inline index.html]: please replace those tabs with spaces. use space bar instead of the tab key to fix indentation.

**Comment by roonyh** [comment]
Thanks! Good job! can you please fix the small nit above?

**Comment by roonyh** [comment]
And your image has to be of square shape. (1:1)

**Comment by hossainalikram** [comment]
Changed the pic and deleted the spaces

**Pull request merged**